tests/SbrpTests/ValidationTests.cs

```diff
@@ -0,0 +1,166 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SbrpTests;
+
+public class ValidationTests
+{
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+    private const string SbrpRepoIdentifier = "source-build-reference-packages";
+    private const string VersionPattern = @"(\.\d)+([\-\w])*";
+    public ITestOutputHelper Output { get; set; }
+
+    public ValidationTests(ITestOutputHelper output) => Output = output;
+
+    [SkippableFact]
+    public void ValidateSbrpAttribute()
+    {
+        Skip.If(RuntimeInformation.IsOSPlatform(OSPlatform.Windows), "This test is not supported on Windows.");
```

Comment:
This is repeated for each test in this class. Would it work to place this just in the class constructor instead? In xUnit, the constructor gets invoked for each test if I remember correctly.

Reply:
I didn't know that it was possible to skip via the constructor, thanks for pointing this out!